### ISSUE TITLE
TH3 and WR1 range limits change

### ISSUE THREAD

**Issue by jlaforga** [shadow-robot/sr-ros-interface#175]
Hi,

I've just realized that the range limits of joints WR1 and TH3 have changed and, speaking with Toni, he says that these changes were made from a Guillaume's suggestion.

To be fair, I think that these changes are good and they are going to improve the hand, but it's not so simple. If these limits change the specifications of the hands have to change as well as another files in 'sr_visualization' (joint_sliders).

I'm going to speak with Armando about it, and see if he is also agreed to change these limits. But by the time being, these limits should move back to the original ones. It means:

WR1 from -40 to -45 and from +28 to +30
TH3 from ±12 to ±15

I will keep you updated as soon as I talk to Armando

Cheers

**Comment by guihomework** [comment]
If I remember well, the problem is that hard-limits (pushing the joint to its limit manually) sometimes do not match controllable limits (limit reachable by pulling on the tendon with reasonable force). Because software limits are too high, controllers continue pulling to reach this limit. 
For instance, xFJ3 can physically move into negative but not through tendon pulling.  It might be the same for THJ3 and WRJ1 (I would need to double check why I suggested these new values some times ago) that the software limits can never be reached without having very large forces (500+ gauge units).

**Comment by jlaforga** [comment]
Yes, I'm agreed that we need to change these limits, but what I'm saying is this is not the right way to do it. Firstly, we need to change the documentation before implement this changes in all of the hands.
